Author: ljlane
Description: nfacct source does not include a changelog.
  This is a compilation of the external changelog files
  from http://netfilter.org/projects/nfacct/files/

--- /dev/null
+++ b/Changelog
@@ -0,0 +1,15 @@
+changes-nfacct-1.0.0.txt:
+
+Pablo Neira Ayuso (11):
+      initial import
+      nfacct: use return instead of exit(EXIT_FAILURE)
+      build: remove bogus dependencies
+      nfacct: check for too many arguments
+      nfacct: check for wrong arguments
+      nfacct: replace one leftover perror by nfacct_perror
+      doc: minor fixes for nfacct.8
+      nfacct: fix licensing terms, this is GPLv2 software
+      nfacct: add XML output support
+      nfacct: update manpage to include xml and reset options
+      build: nfacct release 1.0.0
+
